cmd/monaco/main.go

```diff
@@ -44,380 +36,211 @@ func Run(args []string) int {
 	return RunImpl(args, util.NewFileReader())
 }
 
-func containsVersionFlag(args []string) bool {
-	for _, arg := range args {
-		if arg == "--version" || arg == "-version" {
-			return true
-		}
-	}
-
-	return false
-}
-
 func RunImpl(args []string, fileReader util.FileReader) (statusCode int) {
-	if containsVersionFlag(args) {
-		fmt.Println(version.MonitoringAsCode)
-		return 0
-	}
-
-	statusCode = 0
-
-	dryRun, verbose, environments, projectNameToDeploy, path, errorList, flagError := parseInputCommand(args, fileReader)
-
-	if flagError != nil {
-		util.FailOnError(flagError, "could not parse flags")
-	}
+	var app *cli.App
 
-	var deploymentErrors = make(map[string]error)
-
-	for i, err := range errorList {
-		configIssue := fmt.Sprintf("environmentfile-issue-%d", i)
-		deploymentErrors[configIssue] = err
-	}
-
-	err := util.SetupLogging(verbose)
-
-	if err != nil {
-		util.Log.Error("Error writing log file: %s", err.Error())
+	if newCli, ok := os.LookupEnv("NEW_CLI"); ok && newCli != "0" {
+		app = buildExperimentalCli(fileReader)
+	} else {
+		app = buildCli(fileReader)
```

Comment:
And here something like:

```
You are currently using the old CLI structure which will be used by default until monaco version 2.0.0.
Check out the beta of the new CLI by adding the environment variable `NEW_CLI`. We can't wait for your feedback.
```

Reply:
Should we always display this log message?

The issue I see with this is that this kinda breaks `--version` as it 
now prints more than just the version.

Reply:
But whoever depends on just the version printing can fix their
scripts via `./monaco --version | tail -1`. I guess this is not 
that big of a deal then.

Reply:
Hmm, right! I did not think of the version. Yeah, I guess other tools print more than the version as well. So you would always have to do some extracting/parsing afterwards. E.g. kubectl looks like that:

```
Client Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.2", GitCommit:"66049e3b21efe110454d67df4fa62b08ea79a19b", GitTreeState:"clean", BuildDate:"2019-05-16T16:23:09Z", GoVersion:"go1.12.5", Compiler:"gc", Platform:"linux/amd64"}
The connection to the server 69C15CD5DF601CD3C6660C5439F1C18B.gr7.us-east-1.eks.amazonaws.com:5000 was refused - did you specify the right host o
```

Reply:
Also I used `fmt.Print` instead of `util.Log` to print this info, as I personally think
it adds no value to the log file and is quite annoying to have the logger prefixing
the date and log level before the message.